src/MvcClient.Portal/Controllers/HomeController.cs

```diff
@@ -9,10 +10,12 @@ namespace MvcClient.Portal.Controllers
     public class HomeController : Controller
     {
         private readonly ILogger<HomeController> _logger;
+        private readonly IConfiguration configuration;
 
-        public HomeController(ILogger<HomeController> logger)
+        public HomeController(ILogger<HomeController> logger, IConfiguration configuration)
         {
             _logger = logger;
+            this.configuration = configuration;
         }
```

Comment:
do we need this keyword here, can we have configuration variable starting with underscore to keep consistency please.

Reply:
thanks!